src/components/VAudioDetails/VAudioDetails.vue

```diff
@@ -123,6 +123,8 @@ dt {
 }
 
 dd {
```

Comment:
It's best to use Tailwind here, so that if we update it, this style is updated as well.
You can copy the values from VImageDetails:
https://github.com/WordPress/openverse-frontend/blob/28a6c434ff219e298dcc2b332ad25e1064594e0f/src/components/VImageDetails/VImageDetails.vue#L98-L101

Reply:
Check the last commit, i have fixed it. Couldn't find a taildwind utility class related to font weight so i added that directly.